src/test/java/org/broadinstitute/hellbender/tools/spark/sv/integration/SvDiscoverFromLocalAssemblyContigAlignmentsSparkIntegrationTest.java

```diff
@@ -0,0 +1,152 @@
+                    " -R " + twoBitReference +
+                    " -I " + assemblyBam +
+                    " -O " + outputDir +
+                    (altContigsFile == null ? "" : " -alt-tigs " + altContigsFile) +
+                    (cnvCallsLoc == null ? "" : " --cnv-calls " + cnvCallsLoc);
+        }
+
+        String getOutputDir() {
+            return outputDir;
+        }
+    }
+
+    @DataProvider(name = "svDiscoverFromLocalAssemblyContigAlignmentsSparkIntegrationTest")
+    public Object[][] createTestData() {
+        final File tempDir = GATKBaseTest.createTempDir("symb");
+        tempDir.deleteOnExit();
+
+        final List<Object[]> tests = new ArrayList<>(2);
+        final TestArgs testArgs = new TestArgs(tempDir.getAbsolutePath(), REFERENCE, NON_CANONICAL_CHROMOSOMES,
+                SYMBOLIC_ASSEMBLY_BAM, null);
+        tests.add(new Object[]{testArgs,
+                SYMBOLIC_EXPECTED_SIMPLE_CHIMERA_VCF,
+                SYMBOLIC_EXPECTED_COMPLEX_CHIMERA_VCF,
+                SYMBOLIC_EXPECTED_CPX_RETERINTERPRETATION_1_SEG_VCF,
+                SYMBOLIC_EXPECTED_CPX_RETERINTERPRETATION_MULTI_SEG_VCF,
+                SYMBOLIC_EXPECTED_MERGED_SIMPLE_VCF
+        });
+
+        final File tempDir1 = GATKBaseTest.createTempDir("bnd");
+        tempDir1.deleteOnExit();
+        final TestArgs testArgs1 = new TestArgs(tempDir1.getAbsolutePath(), REFERENCE, NON_CANONICAL_CHROMOSOMES,
+                BND_ASSEMBLY_BAM, null);
+        tests.add(new Object[]{testArgs1,
+                BND_EXPECTED_SIMPLE_CHIMERA_VCF,
+                null,
+                null,
+                null,
+                BND_EXPECTED_MERGED_SIMPLE_VCF
+        });
+
+        return tests.toArray(new Object[][]{});
+    }
+
+    @Test(dataProvider = "svDiscoverFromLocalAssemblyContigAlignmentsSparkIntegrationTest", groups = "sv")
+    public void testDiscoverVariantsRunnableLocal(final TestArgs params, final String expectedSimpleChimeraFilePath,
+                                                  final String expectedComplexFilePath, final String expectedReInterpretationOneSegFilePath, final String expectedReInterPretationMultiSegFilePath,
+                                                  final String expectedMergedSimpleFilePath) throws Exception {
+
+        final List<String> args = Arrays.asList( new ArgumentsBuilder().add(params.getCommandLine()).getArgsArray() );
+        runCommandLine(args);
+
+        final String outputDir = params.getOutputDir();
+
+        final String simpleChimeraVCF = outputDir + TEST_SAMPLE_NAME + "_" + SvDiscoverFromLocalAssemblyContigAlignmentsSpark.SIMPLE_CHIMERA_VCF_FILE_NAME;
+        StructuralVariationDiscoveryPipelineSparkIntegrationTest
+                .svDiscoveryVCFEquivalenceTest(
+                        simpleChimeraVCF,
+                        expectedSimpleChimeraFilePath,
+                        null,
+                        Collections.emptyList(),
+                        false);
+
```

Comment:
I personally found it just slightly complicated to follow the logic of how null expected paths would play out. Maybe worth just putting a comment here that null paths are interpreted as "no variants", so the test will be asserting that the produced variants file is empty.

Reply:
documented `StructuralVariationDiscoveryPipelineSparkIntegrationTest.svDiscoveryVCFEquivalenceTest(...)`